### PR TITLE
chore(rename): repo renaming

### PR DESCRIPTION
**Pull request by janeyuaws**
Issue #, if available: GALI-2443

Description of changes:
* update SOLUTION_NAME to `Research Service Workbench on AWS`
* update links in README etc. to reflect new repo name `research-service-workbench-on-aws`
* update single-click template to `rsw-dev-test`
* update file name (postman and openAPI)
* update stack name to be `rsw-*` and change resource name accordingly

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

* [x] Have you successfully deployed to an AWS account with your changes?
* [x] integ test passed locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,14 @@
 
 # Codecov
 ## `main branch coverage`
-[![codecov](https://codecov.io/github/aws-solutions/solution-spark-on-aws/branch/main/graph/badge.svg?flag=root)](https://app.codecov.io/github/aws-solutions/solution-spark-on-aws/tree/main)
+[![codecov](https://codecov.io/github/aws-solutions/research-service-workbench-on-aws/branch/main/graph/badge.svg?flag=root)](https://app.codecov.io/github/aws-solutions/research-service-workbench-on-aws/tree/main)
 
 ## `develop branch coverage`
-[![codecov](https://codecov.io/github/aws-solutions/solution-spark-on-aws/branch/develop/graph/badge.svg?flag=root)](https://app.codecov.io/github/aws-solutions/solution-spark-on-aws/tree/develop)
+[![codecov](https://codecov.io/github/aws-solutions/research-service-workbench-on-aws/branch/develop/graph/badge.svg?flag=root)](https://app.codecov.io/github/aws-solutions/research-service-workbench-on-aws/tree/develop)
```
Comment (htztomic) [inline README.md:9]: codecov will require an update as well @insignias

Comment (janeyuaws) [reply README.md:9]: Thanks for the heads-up! i will keep that in mind

Comment (insignias) [reply README.md:9]: codecov should be working now !

**Comment by SanketD92** [comment]
More changes are needed to update the main template/stack name from `swb-*` to `rsw-*` (you can search for `swb-$` to find these in the repo). Once done, please verify that template installation is successful.

**Review by SanketD92 — APPROVED** [review]

**Review by SanketD92 — APPROVED** [review]

**Review by SanketD92 — APPROVED** [review]

**Pull request merged**